webapp/channels/src/sass/components/_videos.scss

```diff
@@ -2,72 +2,132 @@
 @use "utils/variables";
 
 .post {
+    // Container for both video and thumbnail
     .video-div {
         position: relative;
-        overflow: hidden;
+        overflow: hidden; // Ensure hover effects stay contained
+        width: 100%;
         max-width: 480px;
-        height: 360px;
-        border-radius: 4px;
         margin-bottom: 8px;
 
-        iframe,
-        video {
+        iframe {
+            position: absolute;
+            top: 0;
+            left: 0;
             display: block;
-            max-width: 100%;
+            overflow: hidden;
+            width: 100%;
+            height: 100%;
+            border: none;
+            border-radius: var(--radius-s);
+            aspect-ratio: 16/9; // Always use 16:9
```

Comment:
Is this safe to change when YouTube shorts are such a big thing now?

Reply:
Ideally we would set the aspect ratio dynamically, but I don't know of a way to get the aspect ratio from the youtube url, so this was what I landed on. 16:9 seemed more common than 4:3 now.